packages/nfts/test/party-ticket/TaikoPartyTicket.t.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import { Test } from "forge-std/src/Test.sol";
+
+import { TaikoPartyTicket } from "../../contracts/party-ticket/TaikoPartyTicket.sol";
+import { Merkle } from "murky/Merkle.sol";
+import { ERC1967Proxy } from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import { MockBlacklist } from "../util/Blacklist.sol";
+
+contract TaikoPartyTicketTest is Test {
```

Comment:
can you test upgrading the implementation, not sure if this is supported by your code.